### DISCUSSION TITLE
Off-Canvas option to change Name in the list for better identification

### DISCUSSION THREAD

**Discussion by k4znIm** [elementor/elementor#28296] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Example: 
I have contact page and show more contact information about specific person after clicking on icon by his name. When setting up icon and selecting option open off-canvas i have to select specific off-canvas what i want to open. 

Problem: 
I have more than one off-canvas and when selecting whitch one i want to open by that icon, all names are like off-canvas1, off-canvas2 ... You have to remember whitch number belong who.

### Describe the solution you'd like.

 By adding option to change name on specific off-canvas this problem would be eliminated.

### Describe alternatives you've considered.

It can also be solved by letting us set id, or class on off-canvas and use that as name when selecting it.

### Additional context

![image](https://github.com/user-attachments/assets/64098679-12c5-4efb-9aef-46b8d1f1498b)

![image](https://github.com/user-attachments/assets/87dd51d8-f32d-466a-98e8-6d54e76a5f11)

![image](https://github.com/user-attachments/assets/9a6cc3a3-3fa1-42e5-a03e-fa0b9355a918)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by k4znIm** [comment]
Thank you for adding this! Now its perfect!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.24.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
